Tests/conf.json

```diff
@@ -3188,6 +3200,456 @@
             "integrations": "FeedExpanse",
             "playbookID": "Feed Expanse Test",
             "fromversion": "6.0.0"
+        },
+        {
+            "playbookID": "Microsoft Graph API - Test",
+            "fromversion": "5.0.0",
+            "integrations": [
+                "Microsoft Graph API"
+            ]
+        },
+        {
+            "playbookID": "BastilleNetworks-Test",
+            "fromversion": "5.0.0",
+            "integrations": [
+                "Bastille Networks"
+            ]
+        },
+        {
+            "playbookID": "AgariPhishingDefense-Test",
+            "fromversion": "5.0.0",
+            "integrations": [
+                "Agari Phishing Defense"
+            ]
+        },
+        {
+            "playbookID": "IronDefense Test",
+            "fromversion": "5.0.0",
+            "integrations": [
+                "IronDefense"
+            ]
+        },
+        {
+            "playbookID": "FeedTalosTestPlaybook",
+            "fromversion": "5.5.0",
+            "integrations": [
+                "Talos Feed"
+            ]
+        },
+        {
+            "playbookID": "Humio-Test",
+            "fromversion": "5.0.0",
+            "integrations": [
+                "Humio"
+            ]
+        },
+        {
+            "playbookID": "Blueliv_ThreatContext_test",
+            "fromversion": "5.0.0",
+            "integrations": [
+                "Blueliv ThreatContext"
+            ]
+        },
+        {
+            "playbookID": "Sophos Central Test",
+            "fromversion": "5.0.0",
+            "integrations": [
+                "Sophos Central"
+            ]
+        },
+        {
+            "playbookID": "Darktrace Test Playbook",
+            "fromversion": "6.0.0",
+            "integrations": [
+                "Darktrace"
+            ]
+        },
+        {
+            "playbookID": "Recorded Future Test Playbook",
+            "fromversion": "5.0.0",
+            "integrations": [
+                "Recorded Future v2"
+            ]
+        },
+        {
+            "playbookID": "X509Certificate Test Playbook",
+            "fromversion": "6.0.0"
+        },
+        {
+            "playbookID": "Acalvio Sample Playbook",
+            "fromversion": "5.0.0",
+            "integrations": [
+                "Acalvio ShadowPlex"
+            ]
+        },
+        {
+            "playbookID": "XMCyberIntegration-Test",
+            "fromversion": "6.0.0",
+            "integrations": [
+                "XMCyber"
+            ]
+        },
+        {
+            "playbookID": "3da36d51-3cdf-4120-882a-cee03b038b89",
+            "fromversion": "5.0.0",
+            "integrations": [
+                "FortiManager"
+            ]
+        },
+        {
+            "playbookID": "Logzio - Test",
```

Comment:
@ShahafBenYakir correct me if im wrong, but we're not supposed to run this test

Reply:
Yes, partner playbooks can be deleted, they will not get picked up automatically.